main.py

```diff
@@ -121,16 +121,18 @@ def choose_direction(data, average):
     for direction, data_surfaces in data_direction_surfaces:
         if all(average_surface in data_surfaces for average_surface in average_surfaces):
             return direction
-    print("None of the csv file contains all surfaces. This just does not work here...")
-    quit()
+    return None
            
 if __name__ == "__main__":
     factors = get_factors()
     files = get_files(data_path, ".csv")
     data = read(files)
     average = process(data, factors)
     the_chosen_one = choose_direction(data, average)
-    files = {file_name: direction for file_name, direction in files.items() if direction == the_chosen_one}
-    for file in files:
-        write(file, average)
+    if the_chosen_one == None:
```

Comment:
```if the_chosen_one == None``` is kinda redundant, if you wish, you can write the statement as ```if not the chosen_one```, since ```None``` object will return ```False``` upon boolean check and non-```None``` object doing the opposite, it is a great way to check whether an object has been instantiated.

Reply:
That said, it's only a preference not a requirement. Resolving the conversation.